assesspy/tests/test_metrics.py

```diff
@@ -38,7 +68,7 @@ def test_metric_succeeds_on_good_input(self, metric, good_input):
 
     def test_metric_met_function_thresholds(self, metric, metric_val):
         if metric == "ki":
-            pt.skip("Skipping test for 'ki' metric (ki_met does not exist)")
+            return None
```

Comment:
We're not actually skipping a test, we just don't even need to run it. Switching to this has the advantage of not showing a "N skipped tests" warning in the `pytest` results.